src/components/IssueQueue.vue

```diff
@@ -157,6 +157,7 @@ watch(params, () => {
         <span class="visually-hidden">Loading...</span>
       </span>
       <span v-if="isLoading"> Loading Flaws&hellip; </span>
+      <span class="float-end fw-bold"> Total: {{ issues.length }}</span>
```

Comment:
I think we should show the number of fetched results (`issues.length`) and the `.count` from the response from GET requests to `https://osidb-stage.prodsec.redhat.com/osidb/api/v1/flaws`

Reply:
https://github.com/RedHatProductSecurity/osim/blob/fix/OSIDB-2390-sequential-saving-on-flaws/src/composables/useFlaws.ts#L17-L63

You can access `response.data.count` in the highlighted block above and use a `ref` to capture the number of results found by OSIDB.

Reply:
Thanks for your feedback. I will update to show both total and loaded count.